packages/core/core/bin.js

```diff
@@ -10,12 +10,18 @@ program
 
```

Comment:
Accidentally kept this file in the rebase, not really relevant to the PR though.